pkg/common/utils/mysql/mysql.go

```diff
@@ -155,6 +172,28 @@ func (db *DB) GetObservers() ([]*Frontend, error) {
 	return res, nil
 }
 
+func (db *DB) GetBackendsByCGName(cgName string) ([]*Backend, error) {
+	backends, err := db.ShowBackends()
+	if err != nil {
+		klog.Errorf("GetBackendsByCGName show backends failed, err: %s\n", err.Error())
+		return nil, err
+	}
+	var res []*Backend
+	for _, be := range backends {
+		var m map[string]interface{}
+		err := json.Unmarshal([]byte(be.Tag), &m)
+		if err != nil {
+			klog.Errorf("GetBackendsByCGName show backends tag get compute_group_name failed, be: %+v, err: %s\n", be, err.Error())
+			return nil, err
+		}
+
+		if fmt.Sprintf("%s", m["compute_group_name"]) == cgName {
```

Comment:
split two step for good reading and debuging, also, for unpredictable situations.